Description: Drop non-Debian platforms
Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

Forwarded: not-needed

We are only interested in building for Debian at the moment, so we do
not need the mach or winapi variants

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -44,16 +44,9 @@
 default = ["use_std"]
 unix_sysv = []
 use_std = ["libc/use_std"]
-[target."cfg(all(any(target_os = \"macos\", target_os = \"ios\"), not(feature = \"unix_sysv\")))".dependencies.mach]
-version = "0.3"
-default-features = false
 [target."cfg(any(unix, target_os = \"dragonfly\"))".dependencies.libc]
 version = "0.2"
 default-features = false
-[target."cfg(target_os = \"windows\")".dependencies.winapi]
-version = "0.3.*"
-features = ["memoryapi", "handleapi", "sysinfoapi", "winbase"]
-default-features = false
 [badges.appveyor]
 repository = "gnzlbg/slice_deque"
 
